src/lib/utils/convertCityToGeoJSON.ts

```diff
@@ -0,0 +1,37 @@
+// Here is a sample GeoJSON document
+// {
+//   "type": "FeatureCollection",
+//     "features": [
+//       {
+//         "type": "Feature",
+//         "geometry": {
+//           "type": "Point",
+//           "coordinates": [102.0, 0.5]
+//         },
+//         "properties": {
+//           "prop0": "value0"
+//         }
+//       },
+// }
+
+export const convertCityToGeoJSON = data => {
+  return {
+    type: "FeatureCollection",
+    features: data.map(({ node }) => {
+      const {
+        coordinates: { lat, lng },
+      } = node.location
+
+      return {
+        type: "Feature",
+        geometry: {
+          type: "Point",
+          coordinates: [lng, lat],
```

Comment:
This is long/lat, which appears to be correct GeoJSON, but our codebase uses lat/long. I'd recommend including a comment so someone doesn't come in later and "fix" this 😄

Reply:
so weird, but yeah - GeoJSON spec denotes the other way around. Weird

Reply:
Yeah this will always be weird until the end of time 

https://macwright.org/lonlat/

😭

Reply:
lol yeah I was getting confused just working on this, it's even more inconsistent than I thought lol